packages/cli/src/ActiveWorkflowRunner.ts

```diff
@@ -348,30 +345,31 @@ export class ActiveWorkflowRunner {
 	/**
 	 * Returns the ids of the currently active workflows
 	 */
-	async getActiveWorkflows(user?: User): Promise<IWorkflowDb[]> {
+	async getActiveWorkflows(user?: User): Promise<string[]> {
 		let activeWorkflows: WorkflowEntity[] = [];
-
 		if (!user || user.globalRole.name === 'owner') {
 			activeWorkflows = await Db.collections.Workflow.find({
 				select: ['id'],
 				where: { active: true },
 			});
+			return activeWorkflows.map((workflow) => workflow.id.toString());
```

Comment:
why is the `.toString` needed? this is already handled in the ORM code, and at this point the `id` should already be a string.

Reply:
for some reason js turns the string into a number again, which lets the tests fail because we get arrays of `[1, '1']` now. so i'm forcing the string

Reply:
this sounds like a bug that should be fixed, instead of being worked around.